skpro/regression/tests/test_ondil.py

```diff
@@ -0,0 +1,48 @@
+import numpy as np
+import pandas as pd
+import pytest
+
+try:
```

Comment:
import `ondil` only in the test, nad add a `pytest.mark.skip` decorator to it, like in other tests dependent on soft depenencies. Search for use of `pytest.mark.skip` in the repo to see examples.